PropertyPrint.java

```diff
@@ -0,0 +1,18 @@
+public class PropertyPrint {
+    public static void main(String[] args) {
+        if(null == args || 0 == args.length) {
+            args = new String[] { "java.version" };
+        }
+
+        for(int i=0; i<args.length; ++i) {
+            String property = args[i];
+
+            if(i > 0) {
+                System.out.print(' ');
+            }
+
+            System.out.print(System.getProperty(property));
+        }
+        System.out.println();
+    }
+}
```

Comment:
I only see one usage of this (`jarch=$( $JAVA_HOME/bin/java -classpath . PropertyPrint os.arch )`); can we simplify this to something like below:

```suggestion
public class PropertyPrint {
    public static void main(String[] args) {
        if(null == args || args.length != 1) {
            System.out.println("Please pass a single argument of which property you want printed.");
            System.exit(1);
        }
        System.out.println(System.getProperty(args[0]));
    }
}
```

Reply:
Let me try to remove it entirely.

Reply:
SGTM!